package.json

```diff
@@ -17,18 +17,21 @@
     "lint:fix": "./node_modules/.bin/eslint . --fix"
   },
   "dependencies": {
-    "antlr4": "~4.6.0"
+    "antlr4": "~4.6.0",
+    "dtrace-provider": "^0.8.7"
```

Comment:
I don't think this should be here.  `dtrace-provider` is a transitive dependency, not a direct dependency -- and putting it here doesn't seem to make the `dtrace-provider` stack traces go away anyway.